README.md

```diff
@@ -1,11 +1,23 @@
 # GOV.UK Connect
 
-This is a command line tool to help you connect to GOV.UK's infrastructure.
+This is a command line tool to help you connect to GOV.UK's infrastructure e.g.
+
+```bash
+govuk-connect ssh -e integration frontend
+```
 
 ## Installation
 
 To install:
 
 ```bash
-gem install govuk-connect
+sudo gem install govuk-connect
```

Comment:
I feel like there should be a Better Way.

Reply:
There is: clone the repo, add it to the path. Now where have I seen that before 🤔

Reply:
`brew install` is the logical step ;-)